controls/SV-204445.rb

```diff
@@ -1,43 +1,40 @@
-    and/or monitoring system trap when there is an unauthorized modification of a configuration item."
+  desc "Unauthorized changes to the baseline configuration could make the system vulnerable to various attacks or allow unauthorized access to the operating system. Changes to operating system configurations can have unintended side effects, some of which may be relevant to security.
+
+Detecting such changes and providing an automated response can help avoid unintended, negative consequences that could ultimately affect the security state of the operating system. The operating system's Information System Security Manager (ISSM)/Information System Security Officer (ISSO) and System Administrators (SAs) must be notified via email and/or monitoring system trap when there is an unauthorized modification of a configuration item."
   desc 'check', 'Verify the operating system routinely checks the baseline configuration for unauthorized changes.
-    Note: A file integrity tool other than Advanced Intrusion Detection Environment (AIDE) may be used, but the tool
-    must be executed at least once per week.
-    Check to see if AIDE is installed on the system with the following command:
-    # yum list installed aide
-    If AIDE is not installed, ask the SA how file integrity checks are performed on the system.
-    Check for the presence of a cron job running daily or weekly on the system that executes AIDE daily to scan for
-    changes to the system baseline. The command used in the example will use a daily occurrence.
-    Check the cron directories for a script file controlling the execution of the file integrity application. For
-    example, if AIDE is installed on the system, use the following command:
-    # ls -al /etc/cron.* | grep aide
-    -rwxr-xr-x 1 root root 29 Nov 22 2015 aide
-    # grep aide /etc/crontab /var/spool/cron/root
-    /etc/crontab: 30 04 * * * root /usr/sbin/aide  --check
-    /var/spool/cron/root: 30 04 * * * /usr/sbin/aide  --check
-    If the file integrity application does not exist, or a script file controlling the execution of the file integrity
-    application does not exist, this is a finding.'
-  desc 'fix', 'Configure the file integrity tool to run automatically on the system at least weekly. The following
-    example output is generic. It will set cron to run AIDE daily, but other file integrity tools may be used:
-    # more /etc/cron.daily/aide
-    #!/bin/bash
-    /usr/sbin/aide --check | /bin/mail -s "$HOSTNAME - Daily aide integrity check run" root@sysname.mil'
+
+Note: A file integrity tool other than Advanced Intrusion Detection Environment (AIDE) may be used, but the tool must be executed at least once per week.
+
+Check for the presence of a cron job running daily or weekly on the system that executes AIDE daily to scan for changes to the system baseline. The command used in the example will use a daily occurrence.
```

Comment:
Monthly jobs should not pass the test. The rule says it should be at least weekly, so a monthly interval should fail by default. 
The new release removed the requirement of checking if `aide` is installed (since that's already done in SV-251705. So we could follow the same approach we did in that control's modification - test if `aide` is the file integrity tool or skip to review manually.

Another approach is to assume the default cron job paths and run the tests with the input.

Reply:
Need to log it as an issue

Reply:
Logged issue #192